tests/tst_qPref.qml

```diff
@@ -1,17 +1,12 @@
 // SPDX-License-Identifier: GPL-2.0
 import QtQuick 2.6
 import QtTest 1.2
-import org.subsurfacedivelog.mobile 1.0
```

Comment:
I'm curious why we no longer need to import org.subsurfacedivelog.mobile... isn't that required?

Reply:
This is because we refer to C++ object not C++ classes. Objects you refer to directly as an extension of the QML language, classes needs an "envelope" with version number etc, so you can use version 1.0, allthough the newest is 2.0